rascal-lsp/src/main/java/org/rascalmpl/vscode/lsp/parametric/InterpretedLanguageContributions.java

```diff
@@ -203,15 +206,19 @@ public InterpretedLanguageContributions(LanguageParameter lang, IBaseTextDocumen
             this.implementer = getFunctionFor(contributions, "implementer");
             var summaryConfig = contributions.thenApply(c -> c.stream()
                     .map(IConstructor.class::cast)
-                    .filter(cons -> cons.getConstructorType().getName().equals("summarizer"))
-                    .findAny()
+                    .filter(cons -> {
+                        var name = cons.getConstructorType().getName();
+                        return name.equals("analyzer") || name.equals("builder");
+                    })
+                    .findAny() // TODO: Check if this should be `findFirst()`
```

Comment:
Maybe we should let `summarizer`  map onto `analyzer` (in the deprecated function) so that we only truly run the `builder` on save, and never on demand.